pkg/cmd/browse/browse.go

```diff
@@ -59,10 +59,18 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 	}
 
 	cmd := &cobra.Command{
-		Long:  "Open the GitHub repository in the web browser.",
-		Short: "Open the repository in the browser",
-		Use:   "browse [<number> | <path> | <commit-SHA>]",
-		Args:  cobra.MaximumNArgs(1),
+		Short: "Open repositories, issues, pull requests, and more in the browser",
+		Long: heredoc.Doc(`
+			Transition from the terminal to the web browser to view and interact with:
+
+			- Issues
+			- Pull requests
+			- Repository content
+			- Repository profile
```

Comment:
Apologies, one more suggestion. I think this should be changed to line up with the wording of the examples 😁 

```suggestion
			- Repository home page
```

Reply:
Done.